src/components/Dashboard/Utilization/UtilizationItem.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Col, Row, SparklineChart } from 'patternfly-react';
+
+import { InlineLoading } from '../../Loading';
+
+export const UtilizationItem = ({ id, title, data, maxY, unit, isLoading, LoadingComponent }) => {
+  const actual = !isLoading && Math.round(data[data.length - 1]);
+
+  const axis = {
+    y: {
+      show: true,
+      padding: {
+        top: 0,
+        bottom: 0,
+      },
+      tick: {
+        count: 3,
+      },
+    },
+    x: {
+      show: false,
+    },
+  };
+  if (maxY) {
+    axis.y.max = maxY;
+  }
+
+  const chartData = {
+    columns: [[unit, ...data.map(val => val.toFixed(1))]],
+    unload: true,
+    type: 'area',
+  };
+
+  return (
+    <div className="kubevirt-utilization__item">
+      <Row>
+        <Col lg={2} md={2} sm={2} xs={2}>
```

Comment:
looking at https://docs.google.com/presentation/d/1kfzD1jyEAIejWFKoKSTJLCetwvmIriLI2UlFbyB-xbg/edit#slide=id.g54d7130601_45_38 - we need to improve behaviour of UtilizationItem when there is not enough space

Reply:
Good catch. Can I deal with that in a separate PR?

There are basically two options
- introduce sort of `isNarrow` property (preferred)
- refactor the component structure and handle it via CSS (too complicated considering the case)

Reply:
Sure, lets fix this in followup